bundle.md

```diff
@@ -1,39 +1,28 @@
-# Bundle
+# Filesystem Bundle
 
 ## Container Format
 
-This section defines a format for encoding a container as a *bundle* - a directory organized in a certain way, and containing all the necessary data and metadata for any compliant runtime to perform all standard operations against it.
+This section defines a format for encoding a container as a *filesystem bundle* - a set of files organized in a certain way, and containing all the necessary data and metadata for any compliant runtime to perform all standard operations against it.
 See also [OS X application bundles](http://en.wikipedia.org/wiki/Bundle_%28OS_X%29) for a similar use of the term *bundle*.
 
-The format does not define distribution.
-In other words, it only specifies how a container must be stored on a local filesystem, for consumption by a runtime.
-It does not specify how to transfer a container between computers, how to discover containers, or assign names or versions to them.
-Any distribution method capable of preserving the original layout of a container, as specified here, is considered compliant.
+The definition of a bundle is only concerned with how a container, and its configuration data, are stored on a local file system so that it can be consumed by a compliant runtime.
+Issues such as distribution, including how to transfer a container between runtimes, assigning names, versioning of bundle, or discovery of bundles are out of scope of this specification.
```

Comment:
To be honest, I have no idea what a section titled "OCI Values" even means and I've been pushing to fix that because some people interpret it as just fluff filler text that's non-normative, while others (as you've hinted) view it as part of our "in scope list of work items".  Because if its our list of work items then when 'c' includes "image auditing" and "cryptographic primitives", where is that in our work?

But I was actually hoping to avoid charter discussions as part of this PR.  I'm ok with removing this sentence **for now** but immediately after the charter is finalized I think we need to add something like it back in so people have the right level of expectations w.r.t. what our goals are.